File: python/medium/how_many_solutions_does_this_quadratic_have.py
# How Many Solutions Does This Quadratic Have?
# https://edabit.com/challenge/o2AKq4xy3nfZabKXL

def solutions(a, b, c):
	d = (b**2) - (4*a*c)
	if d > 0:
		return 2
	elif d == 0:
		return 1
	else:
		return 0

File: javascript/medium/how_many_solutions_does_this_quadratic_have.js
// How Many Solutions Does This Quadratic Have
// https://edabit.com/challenge/x2DEtfqm4RhGcziw9

// b^2 - 4ac

function solutions(a, b, c) {
    d = (b**2) - (4*a*c);
    if (d > 0) {
        return 2;
    } else if (d == 0) {
        return 1;
    } else {
        return 0;
    }
}